js/latoken.js

```diff
@@ -0,0 +1,715 @@
+            let  reaminingAmount =  this.safeFloat(order, 'reaminingAmount');
+            let  timeCreated =  this.safeValue(order, 'timeCreated');
+            let timeFilled = (this.safeValue(order, 'timeFilled') == undefined) ? null : this.safeValue(order, 'timeFilled');
+            result.push({
+            'orderId': orderId,
+            'cliOrdId': cliOrdId,
+            'pairId': pairId,
+            'symbol': symbol,
+            'side': side,
+            'orderType': orderType,
+            'price': price,
+            'amount': amount,
+            'orderStatus': orderStatus,
+            'executedAmount': executedAmount,
+            'reaminingAmount': reaminingAmount,
+            'timeCreated': timeCreated,
+            'timeFilled': timeFilled
+            })
+        }
+        return result;
+    }
+
+    async fetchOrdersByStatus (symbol, status, since = undefined, limit = 100, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        let request = {
+            'status': status,
+        };
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['symbol'] = market['symbol'];
+        }
+        if (limit !== undefined)
+            request['limit'] = limit;
+        let response = await this.privateGetOrderStatus (this.extend (request, params));       
+        let orders = this.parseOrders (response);
+        return orders;
+    }
+
+    async fetchAllActiveOrders(symbol, limit = 50, params = {}) {
+        await this.loadMarkets();
+        let request = {
+            'symbol': symbol,
+            'limit': limit,
+        }
+        request['timestamp'] = this.nonce();
+        let response = await this.privateGetOrderActive(this.extend(request, params));
+        let orders = this.parseOrders(response);
+        return orders;
+    }
+    
+    async fetchOrdersByOrderId(id) {
+        await this.loadMarkets();
+        let request = {
+            'orderId': id,
+        }
+        let response = await this.privateGetOrderGetOrder(request);
+        let result = {};
+        result.orderId = response['orderId'];
+        result.cliOrdId = response['cliOrdId'];
+        result.pairId = this.safeValue(response, 'pairId');
+        result.symbol = response['symbol'];
+        result.side = response['side'];
+        result.orderType = response['orderType'];
+        result.price = this.safeFloat(response, 'price');
+        result.amount = this.safeFloat(response, 'amount');
+        result.orderStatus = response['orderStatus'];
+        result.executedAmount = this.safeFloat(response, 'executedAmount');
+        result.reaminingAmount = this.safeFloat(response, 'reaminingAmount');
+        result.timeCreated = this.safeValue(response, 'timeCreated');
+        result.timeFilled = (this.safeValue(response, 'timeFilled') == undefined) ? null : this.safeValue(response, 'timeFilled');
+        return {
+            'orderId': result.orderId,
```

Comment:
none of these are standardised field names. Please look at other exchange implementations and read https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md thoroughly